core/src/test/java/com/ugamsol/core/models/impl/PriceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package com.ugamsol.core.models.impl;
+
+import com.ugamsol.core.models.BannerArea;
+import com.ugamsol.core.models.Price;
+import io.wcm.testing.mock.aem.junit5.AemContext;
+import io.wcm.testing.mock.aem.junit5.AemContextExtension;
+import org.apache.sling.api.resource.Resource;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith({AemContextExtension.class, MockitoExtension.class})
+class PriceImplTest {
+
+    private final AemContext aemContext = new AemContext();
+    Price price;
+
+    @BeforeEach
+    void setUp() {
+        aemContext.addModelsForClasses(PriceImpl.class);
+        //aemContext.load().json(BannerAreaImpl.class.getResourceAsStream ("com/ugamsol/core/models/impl/Banner.json"),  "/component");
```

Comment:
remove comments

Reply:
i have removed the comments and commited the code.